tests/posix/headers/prj.conf

```diff
@@ -9,6 +9,7 @@ CONFIG_NET_TEST=y
 CONFIG_TEST_RANDOM_GENERATOR=y
 CONFIG_NET_SOCKETS=y
 CONFIG_NETWORKING=y
+CONFIG_NET_SOCKETS_PACKET=y
```

Comment:
Some Kconfig options were added near the latest commit (update to POSIX_NETWORKING or pse51, pse52, ...) because those had minor modifications on the behaviour of POSIX_API.

Previously, we used POSIX_API as a kind of "enable all". The problem was, the feature set kept growing. Not only did binaries get bigger, but some tests would fail to fit into rom / ram. Then I tried to make POSIX_API equal to BASE but too many things broke. Eventually, I determined that generally, 99% of tests and samples would build with the current interpretation, which is actually a fixed set of features.

Reply:
Fixed